public/member_ops.js

```diff
@@ -179,10 +219,91 @@ function handle_members(event) {
             });
         }
       });
+
+    // Create Fact
     } else if (event.target === createFact) {
       // Your code goes here!
+      var dataObject = {
+        member_id: memberId4.value,
+        fact_text: factText.value,
+        likes: likes.value,
+      };
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberId4.value}/facts`, {
+        method: "POST",
+        headers: headers,
+        body: JSON.stringify(dataObject),
+    }).then((response) => {
+      if (response.status === 201) {
+        response.json().then((data) => {
+          resultsDiv.innerHTML = "";
+          let parag = document.createElement("P");
+          parag.textContent = JSON.stringify(data);
+          resultsDiv.appendChild(parag);
+        });
+      } else {
+        response
+          .json()
+          .then((data) => {
+            alert(
+              `Return code ${response.status} ${
+                response.statusText
+              } ${JSON.stringify(data)}`,
+            );
+          })
+          .catch((error) => {
+            console.log(error);
+            alert(error);
+          });
+      }
+    });
+
+    // Update Fact 
     } else if (event.target === updateFact) {
       // Your code goes here!
+      var dataObject = {
+        fact_text: factText2.value,
+        likes: likes2.value,
+      };
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberId5.value}/facts/${factNumber2.value}`, {
```

Comment:
Looks like a similar issue here -- the corresponding fact ID input element has id `fact-number`, which appears to be assigned to `factNumber` at the top of this file. When I changed that, all works great on this one! 🎉